docs/csharp/language-reference/operators/default.md

```diff
@@ -5,6 +5,8 @@ ms.date: 03/13/2020
 f1_keywords:
   - "default_CSharpKeyword"
```

Comment:
We can delete this to avoid duplicate f1_keywords.
But **don't** delete it from `keywords/default.md` for backcompat with older Roslyn versions.